pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/tilbakekreving/TilbakekrevingVedlegg.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.pensjon.etterlatte.maler.tilbakekreving
+
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.createAttachment
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.newText
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingBeloeperSelectors.bruttoTilbakekreving
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingBeloeperSelectors.fradragSkatt
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingBeloeperSelectors.nettoTilbakekreving
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingBeloeperSelectors.renteTillegg
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingInnholdDTOSelectors.summer
+
+@TemplateModelHelpers
+val tilbakekrevingVedlegg = createAttachment<LangBokmalNynorskEnglish, TilbakekrevingInnholdDTO>(
+    title = newText(
+            Bokmal to "Oversikt over feiltutbetalinger",
+            Nynorsk to "Oversikt over feiltutbetalinger",
+            English to "Oversikt over feiltutbetalinger",
+    ),
+    includeSakspart = false
+) {
+    title2 {
+        text(
+            Bokmal to "Oversikt over feiltutbetalinger",
+            Nynorsk to "Oversikt over feiltutbetalinger",
+            English to "Oversikt over feiltutbetalinger",
+        )
+    }
+    paragraph {
+        table(
+            header = {
+                column(1) {
+                    text(
+                        Bokmal to "Beløp som skal kreves tilbake i hele feilutbetalingsperioden",
+                        Nynorsk to "Beløp som skal kreves tilbake i hele feilutbetalingsperioden",
+                        English to "Beløp som skal kreves tilbake i hele feilutbetalingsperioden",
+                    )
+                }
+                column(2) {}
```

Comment:
Er det ikke bedre å sette teksten til `column(3)` for å sikre at det ikke blir klemt sammen?

Reply:
Tabellen skal se slik ut:
![image](https://github.com/navikt/pensjonsbrev/assets/18049279/d3c672f8-b0ce-40fa-8c54-f82047c99b88)

Prøvde å kun ha en column for å se om den bare legger seg over men da kom det bare feilmelding.
Vet du om dette er mulig?

Reply:
Aha, nei, da aner jeg ikke 🤷‍♂️